django/contrib/sessions/backends/cached_db.py

```diff
@@ -51,8 +51,13 @@ def exists(self, session_key):
         )
 
     def save(self, must_create=False):
-        super().save(must_create)
-        self._cache.set(self.cache_key, self._session, self.get_expiry_age())
+        try:
+            super().save(must_create)
+            self._cache.set(self.cache_key, self._session, self.get_expiry_age())
+        except MemoryError as e:
+            import logging
+
+            logging.error(f"Caching error: {e}")
```

Comment:
Unfortunately we can't just log willy-nilly, we have to follow the standard process of getting a named logger and logging to that to allow users of the framework to capture those logs.

I'm not sure myself what the appropriate logger is for this module but nessita or felixx will be able to confirm that.

Reply:
Thank you for the feedback and guidance.

Once I have the information from @felixxm or @nessita, I'll make the necessary updates to the code accordingly.

Reply:
I did talk with Mariusz about this, and he confirmed that:

* we should use a new logger for `django.contrib.sessions`:
```python
import logging

logging.getLogger("django.contrib.sessions")
```
* we should add a release note about the adding of this logger and a documentation entry in `ref/logging`

Reply:
@sulabhkatila Any chance you missed the comment above? I don't see those changes reflected in the current diff.